x-pack/plugins/encrypted_saved_objects/server/saved_objects/get_descriptor_namespace.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ISavedObjectTypeRegistry } from 'kibana/server';
+
+export const getDescriptorNamespace = (
+  typeRegistry: ISavedObjectTypeRegistry,
+  type: string,
+  namespace?: string
+) => {
+  const descriptorNamespace = typeRegistry.isSingleNamespace(type) ? namespace : undefined;
+  return descriptorNamespace === 'default' ? undefined : descriptorNamespace;
+};
```

Comment:
👍 

Should we produce a couple of unit tests for this?

Reply:
s'pose 😄